src/op/op_util.cc

```diff
@@ -208,5 +208,31 @@ Stmt Substitute(Stmt s,
   return ir::Substitute(s, init);
 }
 
+Stmt TransformUpdate(const Stage& stage,
+                     const std::unordered_map<IterVar, Range>& dom_map,
+                     Stmt body,
+                     Stmt update)
+{
+   Expr condition = (make_zero(Int(32)) ==  make_zero(Int(32))); //will try Bool(1) 
```

Comment:
build a ```Array<Expr>``` containing condition,  use https://github.com/dmlc/tvm/blob/master/src/arithmetic/compute_expr.h#L42 to do reduction

Reply:
done, please check